### DISCUSSION TITLE
✅ Sync multiple devices

### DISCUSSION THREAD

**Discussion by kevinrook** [isontheline/pro.webssh.net#132] [Ideas]
is there any way or future plan to enable syncing devices.

so my iPhone & ipad can easily be maintained and both uptodate.

i often find myself needing to use a device to find out i havent added a server to it. and rather than a quick import of settings i have to enter all the details again.

i love the app but i manage many servers and would like them all on webssh

**Comment by isontheline** [comment]
Yes it's [already planned here](https://github.com/isontheline/pro.webssh.net/discussions/43) 📆

I think iCloud will be used because I need to ensure that privacy and security are hold.

**Reply by kevinrook** [reply]
i cant wait!

**Reply by CapableFish** [reply]
Is there any news on when this will be possible?

**Reply by isontheline** [reply]
"Soon" : https://github.com/isontheline/pro.webssh.net/issues/43#issuecomment-782916970

**Comment by isontheline** [comment]
Sync is now available since WebSSH 14.15

**Reply by mbierman** [reply]
Trying now. I’m delighted! Thank you.
